components/rhel/rhel-ose/Vagrantfile

```diff
@@ -1,8 +1,14 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-# The private network IP of the VM. You will use this IP to connect to OpenShift.
-PUBLIC_ADDRESS="10.1.2.2"
+# The top level domain for your VM and the created application routes
+TLD = 'cdk'
```

Comment:
@praveenkumar using just 'cdk' as TLD here. Single names like this are not a good idea either, since they are problematic as well in some cases. Landrush though handles it ok and for the URL and oc login I am using _OPENSHIFT_HOSTNAME _. 

App routes will be nice with <app>-<project>.cdk